Cargo.toml

```
@@ -62,7 +62,7 @@ tempfile = "3"
 parking_lot = "0.11"
 jemallocator = { version = "0.3", features = ['unprefixed_malloc_on_supported_platforms', 'profiling'] }
 
-lettre = "0.9"
+lettre = "0.10.0-alpha.1"
```

lettre uses rustls by default

```suggestion
lettre = { version = "0.10.0-alpha.1", default-features = false, features = ["file-transport", "smtp-transport", "native-tls", "hostname", "builder"] }
```

---

Out of curiosity, does anything break if cargo uses rustls to talk to Mailgun?

---

rustls is fine, although it currently doesn't get tested as much as native-tls in lettre. I just pointed out you probably don't want multiple TLS backends.